python/.gitignore

```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Not a big deal but why are there so many files tracked by the gitignore?

Reply:
I just added a generic Python gitignore: https://github.com/github/gitignore/blob/main/Python.gitignore

Reply:
Cool, gitignores 1 level down enable us not to have tracking at the top level!